config/default.yml

```diff
@@ -142,6 +142,11 @@ Performance/InefficientHashSearch:
   VersionAdded: '0.56'
   Safe: false
 
+Performance/IoReadlines:
+  Description: 'Use `IO.each_line` (`IO#each_line`) instead of `IO.readlines` (`IO#readlines`).'
+  Enabled: true
```

Comment:
I'm concerned about false positives when `readlines` is intentionally used as a useful shorthand. By default disabled (`Enabled: false`) seems better.